docs/staking-and-delegation/delegation.md

```diff
@@ -8,31 +8,31 @@ title: "Delegation"
 
-- A TAO holder, i.e., a delegator, also called a **nominator**, stakes with a subnet validator, making this subnet validator a **delegate** of the nominator. This provides support to the delegate as the delegate's effective stake becomes larger, which increases the delegate's impact on the network.
+- A TAO holder, i.e., a delegator, also called a **nominator**, stakes with a subnet validator, making this subnet validator a **delegate** of the nominator. This supports to the delegate as the delegate's effective stake becomes larger, increasing the delegate's impact on the network.
 - The delegate (the subnet validator) then pools all such delegated stake, along with their own stake, and uses this total stake to perform validation tasks in one or more subnets. Daily staking rewards, in proportion to the total stake of the delegate, are credited to the delegate as a result of such validation tasks.
-- After deducting a percentage for the delegate, these staking rewards are given back to the delegate's nominators. 
+- After a percentage is deducted from the delegate, these staking rewards are given back to the delegate's nominators.
 :::tip Delegate take %
-The default value of the delegate take is 18%. As a delegate you can set your own delegate take % by using the `btcli root set_delegate_take` command. See [Setting your delegate take](#setting-your-delegate-take).
+The default value of the delegate take is 18%. As a delegate, you can set your own delegate take % by using the `btcli root set_delegate_take` command. See [Setting your delegate take](#setting-your-delegate-take).
 :::
 - The above percentage of the staking reward is distributed among the delegate's nominators in proportion to the nominators' staked TAO amount with this delegate.
 
 ## Example
-Consider the below example:
+Consider the example below:
 - A delegate holds their own TAO of 800.
-- Three nominators delegate 30, 70 and 100 TAO to the delegate.
-- The effective stake of the delegate is 1000 TAO (30+70+100 of the delegated TAO plus their own 800 TAO), comprising of 80% of delegate's own and remaining 20% from the nominators.
+- Three nominators delegate 30, 70, and 100 TAO to the delegate.
+- The effective stake of the delegate is 1000 TAO (30+70+100 of the delegated TAO plus their own 800 TAO), comprising of 80% of the delegate's own and the remaining 20% from the nominators.
 
 When the staking dividends are received, the dividends are shared in the following way:
 - The delegate would keep 80% of the dividends, based on their 80% proportion of the total stake (0.8).
-- In addition, the delegate would also keep 18% of the dividends earned on the delegated stake (delegated stake is 20%). This is the delegate take.
+- In addition, the delegate would keep 18% of the dividends earned on the delegated stake (the delegated stake is 20%). This is the delegated stake.
```

Comment:
"delegated stake" → "delegate's take %" 
This is not a typo, but is referring to the 18% delegate take %. Maybe you can add a sentence here like "See the above note on the delegate take %".